gnome-image-installer/pages/confirm/gis-confirm-page.c

```diff
@@ -0,0 +1,281 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*- */
+/*
+ * Copyright © 2018 Endless Mobile, Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
+ *
+ */
+
+/* Confirm page {{{1 */
+
+#define PAGE_ID "confirm"
+
+#include "config.h"
+#include "confirm-resources.h"
+#include "gis-confirm-page.h"
+#include "gis-store.h"
+
+#include <udisks/udisks.h>
+#include <glib/gstdio.h>
+#include <glib/gi18n.h>
+#include <gio/gio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+struct _GisConfirmPagePrivate {
+    GtkLabel *vendor_label;
+    GtkLabel *product_label;
+
+    GtkLabel *image1_label;
+    GtkLabel *model1_label;
+    GtkLabel *device1_label;
+    GtkLabel *size1_label;
+
+    GtkBox *warning_box;
+    GtkLabel *warning_label;
+
+    GtkBox *wrong_computer_box;
+
+    GtkLabel *countdown_label;
+    GtkButton *cancel_button;
+    GtkButton *continue_button;
+
+    guint countdown_source;
+    guint countdown_remaining;
```

Comment:
s/`countdown_remaining`/`countdown_remaining_seconds`/ to make the units more obvious?